src/iso_alloc.c

```diff
@@ -490,7 +490,8 @@ INTERNAL_HIDDEN iso_alloc_zone_t *_iso_new_zone(size_t size, bool internal, int3
         create_guard_page(p + g_page_size + tag_mapping_size);
         new_zone->user_pages_start = (p + g_page_size + tag_mapping_size + g_page_size);
         uint64_t *_mtp = p + g_page_size;
-        int64_t tms = tag_mapping_size / sizeof(uint64_t);
+        /* (>> 3) == sizeof(uint64_t) == 8 */
```

Comment:
Wait, the compile is too dumb to optimize `tag_mapping_size / sizeof(uint64_t);` into  ` tag_mapping_size >> 3;`?!

Reply:
I've gotten into the habit of assuming it will not always optimize these division calls. In this case it likely will because sizeof(uint64_t) is known at compile time.